content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Install the AWS Integration
+
+As a first step, make sure to [install the AWS integration][1] before proceeding.
+
+Next, depending on your language and configuration, choose between using the Datadog Lambda Library or Amazon X-Ray for your traces.
```

Comment:
We can add a sentence after here saying something like: if you are looking to trace across functions and hosts, you need to set up both.

Reply:
and we can link to this picture to explain what we mean

https://datadog-docs.imgix.net/images/integrations/amazon_lambda/lambda_tracing.cf509d6b0ca3b5dabb86dbaf7a13f906.png?fit=max&auto=format